images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -291,6 +291,61 @@ meteringconfig_storage_s3Compatible_create_secret: "{{ _storage_spec | json_quer
 meteringconfig_storage_s3Compatible_credentials_secret_name: "{{ _storage_spec | json_query('hive.s3Compatible.secretName') }}"
 meteringconfig_storage_s3Compatible_endpoint: "{{ _storage_spec | json_query('hive.s3Compatible.endpoint') }}"
 
+#
+# Override the default values to handle Openshift, and non-Openshift configurations
+#
+meteringconfig_ocp_disabled: "{{ lookup('env', 'DISABLE_OCP_FEATURES') | bool or meteringconfig_default_values | combine(meteringconfig_spec_overrides, recursive=True) | json_query('disableOCPFeatures.enabled') }}"
```

Comment:
`json_query('disableOCPFeatures.enabled')` should be `json_query('disableOCPFeatures')`